### DISCUSSION TITLE
LexicalNested Composer

### DISCUSSION THREAD

**Discussion by alessandroias** [facebook/lexical#4407] [Q&A]
I am trying to create a custom Note component which displays a User Choice on a Quiz, which a tutor can use to explain why the given answer is incorrect. 
I would like to allow the original quiz text to be formatted, therefore im using nested components. 
Sadly however I get this error: `TypeError: Cannot set properties of undefined (setting 'theme')` when using `LexicalNestedComposer`.
Even if I define an initial Theme the problem persists. Likewise, it doesn't go away even if I don't define a theme at all in the editor.
I can't figure out why this happens.

I Created a `ChoiceNode` which decorates with a `ChoiceComponent` which then renders the `LexicalNestedComponent` and all the additional Choice Context...

If you need more information to help me I won't hesitate to let you have it.
Thank you!

**Comment by Othman-dev** [comment]
Hi,

I don't know if you find your solution but in your case. I would make sure that
`<LexicalComposer initialConfig={initialConfig}>`
(taken from App.tsx from playground)

Receives an initial configuration which could look like this
`const initialConfig = {
    editorState: isCollab
      ? null
      : emptyEditor
      ? undefined
      : prepopulatedRichText,
    namespace: 'Playground',
    nodes: [...PlaygroundNodes],
    onError: (error: Error) => {
      throw error;
    },
    theme: PlaygroundEditorTheme,
  };`
(taken from App.tsx from playground)

And then have a look at 
`<LexicalNestedComposer initialEditor={caption}>`
(taken from ImageComponent.tsx from playground)

Where the initial editor is a Lexical Editor.

You can actual change the theme of a Nested Composer, if that is what you are looking for as well
`<LexicalNestedComposer
      initialEditor={cellEditor}
      initialTheme={cellEditorConfig.theme}
      initialNodes={cellEditorConfig.nodes}
      skipCollabChecks={true}>`
(taken from TableComponent.tsx from playground)

Hoping it help someone.
